### ISSUE TITLE
Experiencing Font Awesome issue in IE11

### ISSUE THREAD

**Issue by nickjer** [OSC/ood-dashboard#305]
This is what the Dashboard looks like in IE11 on Windows 10:

![image](https://user-images.githubusercontent.com/4260509/34215066-4795d6ba-e572-11e7-94c7-8b0314b30304.png)

It downloaded the fonts just fine, but the Console gives the following errors:

![image](https://user-images.githubusercontent.com/4260509/34215086-5937fd62-e572-11e7-9c8e-877a82e837c8.png)

**Comment by nickjer** [comment]
I am using IE11.125.16299.0

**Comment by nickjer** [comment]
May need to walk through this:

https://github.com/FortAwesome/Font-Awesome/wiki/Troubleshooting#im-hosting-fonts-on-my-server-and-icons-dont-show-up

**Comment by brianmcmichael** [comment]
No problems on my end. This is likely related to your internet security settings in Windows.

![untitled](https://user-images.githubusercontent.com/2374718/34215843-a3ec1080-e574-11e7-9aba-e81cd4691d91.png)

**Comment by brianmcmichael** [comment]
See the first answer here:

https://stackoverflow.com/questions/31745307/ie-11-sometimes-preventing-the-display-of-font-awesome-webfonts

**Comment by nickjer** [comment]
Strange, setting security settings to lowest still doesn't work.

**Comment by nickjer** [comment]
I managed to fix it by following the directions here:

https://stackoverflow.com/questions/33023461/ie-11-error-css3111-in-my-own-code-and-google-com-fonts-doesnt-render-any-fon#answer-34106581

In particular this snippet:

> **To disable Untrusted Font Blocking using Registry Editor:**
>
> Open Registry Editor (regedit.exe) and go to the following registry subkey: `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Kernel\`
If the `MitigationOptions` key is not there, right-click and add a new QWORD (64-bit) Value, naming it as `MitigationOptions`.
To turn this feature off. Type `2000000000000`.

Mine was originally set as `1000000000000`. I changed it to the recommended value above and restarted my machine. It worked after the restart.
